### PR TITLE
GitHub actions orchestrator example

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/github_actions_orchestration/README.md [hunk 1]
```diff
@@ -1,73 +1,141 @@
...
 
-## 👣 Step-by-Step
-### 📄 Prerequisites 
+## 📄 Prerequisites
 
-In order to run this example, you need to install and initialize ZenML.
+In order to run your ZenML pipelines using GitHub Actions, we need to set up a few things first:
+
+* First you'll need a [GitHub](https://github.com) account and a cloned repository.
+* You'll also need to create a GitHub personal access token that allows you read/write GitHub secrets and push docker images to your GitHub container registry. To do so, please follow [this guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and make sure to assign your token the **repo** and **write:packages** scopes.
```
Comment (strickvl) [inline examples/github_actions_orchestration/README.md:12]:
```suggestion
* You'll also need to create a GitHub personal access token that allows you read/write GitHub secrets and push Docker images to your GitHub container registry. To do so, please follow [this guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and make sure to assign your token the **repo** and **write:packages** scopes.
```

File: examples/github_actions_orchestration/README.md [hunk 2]
```diff
@@ -1,73 +1,141 @@
...
 
-# Register one of the three following artifact stores and a secret to connect to it
-# 1) AWS
-zenml secret register s3_store_auth --schema=aws --aws_access_key_id=<ACCESS_KEY_ID> --aws_secret_access_key=<SECRET_ACCESS_KEY>
-zenml artifact-store register cloud_artifact_store --flavor=s3 --path=<YOUR_S3_BUCKET_PATH> --authentication_secret=s3_store_auth
-
-# 2) GCP
-zenml secret register gcp_store_auth --schema=gcp ...
-zenml artifact-store register cloud_artifact_store --flavor=gcp --path=<YOUR_GCP_BUCKET_PATH> --authentication_secret=gcp_store_auth
+# Login to the GitHub container registry so we can push the docker images required to run your ZenML pipeline.
```
Comment (strickvl) [inline examples/github_actions_orchestration/README.md:36]:
```suggestion
# Login to the GitHub container registry so we can push the Docker images required to run your ZenML pipeline.
```

File: examples/github_actions_orchestration/README.md [hunk 3]
```diff
@@ -1,73 +1,141 @@
...
+# Now that the stack is active, we can register the secrets needed to connect to our metadata and artifact store:
+zenml secret register mysql_secret \
+    --schema=mysql \
+    --user=<USERNAME> \
+    --password=<PASSWORD> \
+    --ssl_ca=@<PATH_TO_SSL_SERVER_CERTIFICATE> \
+    --ssl_cert=@<PATH_TO_SSL_CLIENT_CERTIFICATE> \
+    --ssl_key=@<PATH_TO_SSL_CLIENT_KEY>
+
+# Register one of the following secrets depending on the flavor of artifact store that you've registered
```
Comment (strickvl) [inline examples/github_actions_orchestration/README.md:106]:
```suggestion
# Register one of the following secrets depending on the flavor of artifact store that you've registered:
```

File: examples/github_actions_orchestration/README.md [hunk 4]
```diff
@@ -1,73 +1,141 @@
...
-### 📆 Run or schedule the pipeline
+## ▶️ Run the pipeline
+
+Now that we're done with all the tedious setup, it's time to run the example pipeline. To do so, simply call
 
 ```bash
 python run.py
 ```
 
+That's it! If everything went as planned this pipeline should now be running in
```
Comment (strickvl) [inline examples/github_actions_orchestration/README.md:134]:
```suggestion
That's it! If everything went as planned, this pipeline should now be running in
```

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by strickvl — APPROVED** [review]
Looks good to me, pending resolution of conflicts etc..

**Review by AlexejPenner — APPROVED** [review]
LGTM

File: examples/github_actions_orchestration/README.md [hunk 5]
```diff
@@ -125,7 +125,18 @@ zenml secret register azure_store_auth \
...
 ## ▶️ Run the pipeline
 
-Now that we're done with all the tedious setup, it's time to run the example pipeline. To do so, simply call
+We're almost done now, but there is one additional step we need to do after our first pipeline ran (and failed). To do so, simply call
+
+```bash
+python run.py
+```
+
+Running your first pipeline using the ZenML GitHubActions orchestrator will create a [GitHub package](https://github.com/features/packages) called **zenml-github-actions** which by default won't be accessible by GitHub Actions.
```
Comment (strickvl) [inline examples/github_actions_orchestration/README.md:134]:
```suggestion
Running your first pipeline using the ZenML GitHub Actions orchestrator will create a [GitHub package](https://github.com/features/packages) called **zenml-github-actions** which by default won't be accessible by GitHub Actions.
```

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by strickvl — APPROVED** [review]

**Pull request merged**